Lecture6(GithubAPI)/components/BodyComponent.js

```diff
@@ -1,10 +1,11 @@
-import { useState, useEffect } from "react";
+import { useState, useEffect, useContext } from "react";
 import Skeleton from "react-loading-skeleton";
 import "react-loading-skeleton/dist/skeleton.css";
```

Comment:
looks different! what is react-skeleton?

Reply:
the loader while you on scrolling on youtube page